### ISSUE TITLE
BUG: Unable to create project when using `litmusctl` with latest (`v3.0.0`) version of Litmus

### ISSUE THREAD

**Issue by smitthakkar96** [litmuschaos/litmusctl#161]
When trying to create a project using litmusctl we get following error:

``` shell
Unmatched status code:{"error":"invalid_request","errorDescription":"The request is missing a required parameter, includes an invalid parameter value, includes a parameter more than once, or is otherwise malformed"}
```

When checking the logs for auth-server I see this in the logs:

``` json
{"file":"/go/pkg/mod/github.com/sirupsen/logrus@v1.4.2/logger.go:192","func":"github.com/sirupsen/logrus.(*Logger).Log","level":"warning","msg":"json: unknown field \"project_name\"","time":"2023-10-16T11:28:38Z"}
```

This is because the `project_name` param has been renamed [here](https://github.com/litmuschaos/litmus/blob/master/chaoscenter/authentication/pkg/entities/project.go#L49C48-L49C59) to `projectName`. Changing payload to use `projectName` fixes the problem. Here is an example `curl` request.

``` bash
curl --location 'http://localhost:9003/create_project' \
--header 'Content-Type: application/json' \
--header 'Authorization: Bearer <JWT TOKEN>' \
--data '{
    "projectName": "awesome-project"
}'
```

**Comment by Nageshbansal** [comment]
Thanks @smitthakkar96 for pointing out this issue. This issue has already been addressed through PR #156 ( and  i think it's going to be merged soon)

**Comment by SarthakJain26** [comment]
Fixed with PR: https://github.com/litmuschaos/litmusctl/pull/156
